proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -115,6 +116,8 @@ enum EventType {
   EVENT_TYPE_USER_AUTO_MERGED_FROM_NEW_IDP = 37;
   EVENT_TYPE_DEVICE_AUTHORIZATION_GRANT_APPROVED = 41;
   EVENT_TYPE_DEVICE_AUTHORIZATION_GRANT_DENIED = 42;
+  EVENT_TYPE_CHECK_PLUGIN_CREATED = 43;
```

Comment:
Why wouldn't this be a "plugin created" and then on the payload plugin created you have a "type" for the plugin?

Reply:
updated, we will be reusing the resource and event type of alpha plugin then